package.json

```diff
@@ -70,6 +70,7 @@
     "eslint-plugin-no-unsanitized": "3.1.4",
     "eslint-plugin-prettier": "3.1.4",
     "eslint-plugin-react": "7.21.5",
+    "eslint-plugin-rulesdir": "^0.1.0",
```

Comment:
pin this?

Reply:
*sigh*, now I need to add a lint check for this. Or I wonder if we can configure yarn to just do this automatically...